metrics/interface/src/main/assembly/resources/conf/iotdb-metric.yml

```diff
@@ -18,11 +18,12 @@
 #
 
 # whether enable the module
-enableMetric: false
+enableMetric: true
 
 # can be multiple reporter, e.g., jmx, prometheus
 metricReporterList:
   - jmx
+  - prometheus
```

Comment:
From my perspective, Default value(jmx) is enough, so you can remove prometheus here

Reply:
I think prometheus is more useful than JMX to DBA or SRE, from which iotdb can easily integrate with prometheus (the standard monitor resolutions of CNCF), and users can get status from a simple ‘curl’ command conveniently.

Reply:
ok, I get it. I think this is reasonable